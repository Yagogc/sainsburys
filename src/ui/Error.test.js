import React from "react";
import renderer from "react-test-renderer";
import { ErrorMsg } from "./Error";

test("UI - ErrorMsg:", () => {
  const tree = renderer.create(<ErrorMsg />).toJSON();
  expect(tree).toMatchSnapshot();
});
