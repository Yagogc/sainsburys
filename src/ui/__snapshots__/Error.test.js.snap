// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI - ErrorMsg: 1`] = `
.c0 {
  width: 100%;
  max-width: 800px;
  padding: 10px;
  margin: 0 auto;
  text-align: center;
  background: red;
  color: white;
  box-shadow: 0 1px 3px rgba(0,0,0,0.12),0 1px 2px rgba(0,0,0,0.24);
  border-radius: 5px;
}

<p
  className="c0"
/>
`;
